Python/omevv/v1/scripts/CreateRepoProfile.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import sys
+import time
+import warnings
+from webbrowser import get
+
+import requests,json,base64
```

Comment:
Since you are using packages not included in standard python installation. Please add a requirements.txt file with package dependencies that can be used by the user. This is also a request made to @ramya-dell . Please collaborate with her for the same.